src/main/java/com/amazon/dlic/util/SettingsBasedSSLConfigurator.java

```diff
@@ -414,7 +414,7 @@ public SSLConfig(SSLContext sslContext, String[] supportedProtocols, String[] su
             this.effectiveKeyAlias = effectiveKeyAlias;
 
             if (log.isDebugEnabled()) {
```

Comment:
do we need isDebugEnabled variable here?

Reply:
This is not reused or loop in same method scope